tests/fixtures/json_content_99999.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+
+from collections import OrderedDict
+
+EXPECTED = OrderedDict([
+    ('id', u'99999'),
+    ('title', u'Fishing for errors in the\xa0tests'),
+    ('impactStatement', u'Testing a document which mimics the format of a file we’ve used  before plus CO<sub>2</sub> and Ca<sup>2+</sup>.'),
+    ('published', '2018-08-01T00:00:00Z'),
+    ('image', OrderedDict([
+        ('thumbnail', OrderedDict([
+            ('uri', 'https://iiif.elifesciences.org/digests/99999%2Fdigest-99999.jpg'),
+            ('alt', ''),
+            ('source', OrderedDict([
+                ('mediaType', 'image/jpeg'),
+                ('uri', 'https://iiif.elifesciences.org/digests/99999%2Fdigest-99999.jpg/full/full/0/default.jpg'),
+                ('filename', 'digest-99999.jpg'),
+                ])),
+            ('size', OrderedDict([
```

Comment:
this fixture seems to contradict the `it will never have width: None` tenet, may be out of date?

Reply:
I have edited this example `width` and `height` to reflect a value different from 0, since IIIF should never return that